src/server/custom-functions/metadata.parser.ts

```diff
@@ -57,15 +57,23 @@ function traverseAST(sourceFile: ts.SourceFile): ICFFunctionMetadata[] {
           if (isCF) {
             const jsDocParamInfo = getJSDocParams(func);
 
-            const parameters = func.parameters.map((p: ts.ParameterDeclaration) => {
-              const name = (p.name as ts.Identifier).text;
+            const [lastParameter] = func.parameters.slice(-1);
```

Comment:
What if there were 0 parameters?

Reply:
Also, style rules should force you to add spaces after `[` and before `]`

Reply:
Agreed, but prettier doesn't seem to think so :/